boards/arm/hexiwear_k64/doc/hexiwear_k64.rst

```diff
@@ -100,6 +100,12 @@ The K64F SoC has five pairs of pinmux/gpio controllers.
 +-------+-----------------+---------------------------+
 | Name  | Function        | Usage                     |
 +=======+=================+===========================+
+| PTA12 | GPIO            | Click 1 I/O button/LED    |
++-------+-----------------+---------------------------+
+| PTA13 | GPIO            | Click 2 I/O button/LED    |
++-------+-----------------+---------------------------+
+| PTA14 | GPIO            | Click 3 I/O button/LED    |
++-------+-----------------+---------------------------+
```

Comment:
These changes should be in the commit where you added the gpios to the board, not in the commit adding a new demo application